tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -258,6 +258,8 @@ def _time_test(self, test_name, data):
     def suite_start(self, data):
         if self.start_timestamp_seconds is None:
             self.start_timestamp_seconds = self._get_time(data)
+        if 'run_info' in data:
+            self.flag_specific = data['run_info'].get('flag_specific') or ''
```

Comment:
```suggestion
            self.flag_specific = data['run_info'].get('flag_specific', '')
```

Reply:
Currently if data['run_info'].get('flag_specific') was equal to something falsey like `0`, or `[]`, `data['run_info'].get('flag_specific') or ''` would evaluate to `''`.